tbtadm/controller.cpp

```diff
@@ -415,26 +442,38 @@ void tbtadm::Controller::printTree(
     for (const auto& device : map)
     {
         auto last = device.first == map.rbegin()->first;
-        m_out << indentation << "|\n";
-        m_out << indentation << "+- " << device.second.m_desc[0];
+        m_out << indentation << symbols[SYMBOL_PIPE] << std::endl;
```

Comment:
I tend to avoid `std::endl` because it forces flush.
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rio-endl